server/activity.go

```diff
@@ -65,8 +65,8 @@ func (s *Server) registerActivityRoutes(g *echo.Group) {
 			}
 			activityFind.CreatorID = &creatorID
 		}
-		if typePrefixStr := c.QueryParams().Get("typePrefix"); typePrefixStr != "" {
-			activityFind.TypePrefix = &typePrefixStr
+		if typePrefixList := c.QueryParams()["typePrefix"]; typePrefixList != nil {
```

Comment:
How is the list passed in as a query parameter?

Reply:
Get `?typePrefix=bb.member&typePrefix=bb.project` -> Parsed `typePrefixList: ["bb.member", "bb.project"]`
Get `?typePrefix=bb.member` -> Parsed `typePrefixList: ["bb.member"]`

Reply:
SGTM, just need to document it so that the list is an OR list instead of AND list.